aws-iot-mitigationaction/packaging_additional_published_artifacts/contract_test_dependencies.yml

```diff
@@ -0,0 +1,46 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: 'Resources for running the AccountAuditConfiguration contract tests'
```

Comment:
This Description seems wrong. Should say "Mitigation Actions contract tests"

Reply:
Fixed